docs/content/developers/release-management.md

```diff
@@ -22,13 +22,14 @@
 <!-- markdown-link-check-disable-next-line -->
 The following “Repository secret” environment variables must be added to <https://github.com/ddev/ddev/settings/secrets/actions>:
 
+* `AMPLITUDE_API_KEY`: Key that enables Amplitude reporting.
 * `AUR_SSH_PRIVATE_KEY`: Private ssh key for the `ddev-releaser` user. This must be processed into a single line, for example, `perl -p -e 's/\n/<SPLIT>/' ~/.ssh/id_rsa_ddev_releaser| pbcopy`.
 * `CHOCOLATEY_API_KEY`: API key for Chocolatey.
 * `DDEV_GITHUB_TOKEN`: GitHub token that gives access to create releases and push to the Homebrew repositories.
 * `DDEV_MACOS_APP_PASSWORD`: Password used for notarization, see [signing_tools](https://github.com/ddev/signing_tools).
 * `DDEV_MACOS_SIGNING_PASSWORD`: Password for the macOS signing key, see [signing_tools](https://github.com/ddev/signing_tools).
 * `DDEV_WINDOWS_SIGNING_PASSWORD`: Windows signing password.
-* `SegmentKey`: Key that enables Segment reporting.
+* `SEGMENTKEY`: Key that enables Segment reporting.
```

Comment:
The name of this environment variable is SegmentKey still. But I'm not convinced we shouldn't be removing this line completely.

Reply:
Oh that's the name of the env var? Thought this is the secret used at GH... I would not remove it at the moment. Removal is planed with a second PR to remove the old implementation once we are happy with the new data which we know after the app was running in the field for some time.

Reply:
Please check the introduction before the list, it's about the secrets and not the env vars.

Reply:
Documented now, we have to stick to the name used in the source.